### DISCUSSION TITLE
ACP-131:  Activate Cancun EIPs on C-Chain and Subnet-EVM chains

### DISCUSSION THREAD

**Discussion by avalanche-foundation-admin** [avalanche-foundation/ACPs#139] [Discussions]
Proposes enable new EVM opcodes and opcode changes in accordance with the following EIPs on the Avalanche C-Chain and Subnet-EVM chains:

[EIP-4844: BLOBHASH opcode](https://eips.ethereum.org/EIPS/eip-4844)
[EIP-7516: BLOBBASEFEE opcode](https://eips.ethereum.org/EIPS/eip-7516)
[EIP-1153: Transient storage](https://eips.ethereum.org/EIPS/eip-1153)
[EIP-5656: MCOPY opcode](https://eips.ethereum.org/EIPS/eip-5656)
[EIP-6780: SELFDESTRUCT only in same transaction](https://eips.ethereum.org/EIPS/eip-6780)

Note blob transactions from EIP-4844 are excluded and blocks containing them will still be considered invalid.

https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/131-cancun-eips

**Comment by thomasnft** [comment]
Is this ACP is available? I'm interested in utilizing EIP-4844 in my project.

**Comment by vittominacori** [comment]
@avalanche-foundation-admin any update about this ACP?

**Comment by xenide** [comment]
has the c-chain been updated to cancun yet?
